lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc

```diff
@@ -183,12 +183,22 @@ function dosomething_signup_sms_game_form($form, &$form_state, $node, $num_frien
     ),
   );
   for ($i = 0; $i < $num_friends; $i++) {
-    $form['group1']['beta_first_name_' .$i] = array(
-      '#type' => 'hidden',
-      '#title' => 'Your Friend\'s Name',
-    );
+    // If this is a multiplayer game, add first name fields.
+    if ($form['sms_game_type']['#default_value'] === 'multi_player') {
+      $form['beta_first_name_' .$i] = array(
+        '#type' => 'textfield',
+        '#title' => 'Your Friend\'s Name',
+        '#required' => TRUE,
+        '#attributes' => array(
+          'data-validate' => 'friend_name',
+          'data-validate-required' => '',
+          'placeholder' => t('(555) 555-5555'),
```

Comment:
That's the grayish placeholder text that shows up on the form. (Or am I missing something else that's wrong haha? :dizzy_face:)